pkg/operator/collection.go

```diff
@@ -484,7 +484,7 @@ func (r *collectionReconciler) makeCollectorConfig(ctx context.Context, spec *mo
 	// Mark status updates in batch with single timestamp.
 	for _, nm := range NodeMons.Items {
 		if spec.KubeletScraping != nil && nm.Namespace == DefaultOperatorNamespace && (nm.Name == reservedKubeletJobName || nm.Name == reservedCAdvisorJobName) {
-			logger.Info("NodeMonitoring job %s was not applied because OperatorConfig.collector.kubeletScraping is enabled. kubeletScraping already includes the metrics in this job.", nm.Name)
+			logger.Info("NodeMonitoring job %s was not applied because OperatorConfig.collector.kubeletScraping is enabled. kubeletScraping already includes the metrics in this job.", "name", nm.Name)
```

Comment:
```suggestion
			logger.Info("NodeMonitoring job was not applied because OperatorConfig.collector.kubeletScraping is enabled. kubeletScraping already includes the metrics in this job.", "name", nm.Name)
```